src/access/v1.ts

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
Do we really need this file? We've given a bit more thought to this API and are still marketing this as a preview release (i.e. the big 1.1 is later in the year), so there's less worry here about breaking backwards compatibility.

(Plus, the "v1" isn't even exposed to developers, so it doesn't actually do anything to help ease breaking changes :) )

Reply:
True, initially I wanted to isolate the export of all the functions, and named it `v1` even if the name wasn't exposed to developers to be consistent with the other files exporting groups of functions in `acp` in particular, but you're right actually this doesn't add any value in this context :)

Reply:
b7725797b5be63c137b0ec84258765b2c756e662